packages/esm-appointments-app/src/appointments/common-components/appointments-table.component.tsx

```diff
@@ -69,6 +69,10 @@ const AppointmentsTable: React.FC<AppointmentsTableProps> = ({
       header: t('identifier', 'Identifier'),
       key: 'identifier',
     },
+    {
+      header: t('location', 'Location'),
+      key: 'location',
+    },
```

Comment:
The location column was present in the Today's view but not the Appointments view, now it is in both.  I noticed that in the Appointments view, we *don't* currently filter by session location?  Not sure if the intent was to filter in that view, and therefore the location column would be redundant?  If we do implement that, we can just configure this table to hide this column as needed.